mldsa/poly.h

```diff
@@ -307,14 +307,14 @@ __contract__(
  * Arguments:   - const mld_poly *a: pointer to polynomial
  *              - int32_t B: norm bound
  *
- * Returns 0 if norm is strictly smaller than B <= (MLDSA_Q-1)/8 and 0xFFFFFFFF
- * otherwise.
+ * Returns 0 if norm is strictly smaller than
+ * B <= (MLDSA_Q - REDUCE32_RANGE_MAX) and 0xFFFFFFFF otherwise.
```

Comment:
Nit
```suggestion
 * Returns 0 if norm is strictly smaller than B, and 0xFFFFFFFF otherwise.
 *
 * B must not be larger than MLDSA_Q - REDUCE32_RANGE_MAX.
```

Reply:
changed.